mods/ts/weapons/missiles.yaml

```diff
@@ -294,3 +294,30 @@ RedEye2:
 	Warhead@2Eff: CreateEffect
 		Explosions: tiny_twlt
 		ImpactSounds: expnew05.aud
+
+QuadLauncher:
+	ReloadDelay: 180
+	Range: 7c0
+	Report: samshot1.aud
+	ValidTargets: Air
```

Comment:
Cyborg reapers were also able to attack ground targets, see http://cnc.wikia.com/wiki/Reaper_(Firestorm) , the Ingame paragraph.